example/cumulus-tf/browse_example.tf

```diff
@@ -0,0 +1,335 @@
+module "discover_granules_browse_example_workflow" {
```

Comment:
Are we wrapping these as modules so that they're not actually included in our deployed resources?

Reply:
Hmm, no, this *will* result in us deploying this workflow set.  

I'm not sure if that's a bad thing or not :thinking face: 

We can move these to another 'example'....uh..... 'example' target directory to avoid that.

Reply:
We were deploying this in Kes world, so it's fine to deploy it via TF as well

https://github.com/nasa/cumulus/blob/master/example/app/config.yml#L134